src/v/pandaproxy/client/client.h

```diff
@@ -132,6 +133,12 @@ class client {
       const kafka::member_id& m_id,
       std::vector<kafka::offset_commit_request_topic> topics);
 
+    ss::future<kafka::fetch_response> consume(
+      const kafka::group_id& g_id,
+      const kafka::member_id& m_id,
+      std::chrono::milliseconds timeout,
+      int32_t max_bytes);
```

Comment:
is int the right type here? i.e.: does it have a sentinel value of `-1` meaning something special?